### ISSUE TITLE
Brook Sahagin aggros through sneak/invis/deodorize

### ISSUE THREAD

**Issue by zierk** [OmegaPrivateServer/issues#279]
## Summary
The Brook Sahagin right inside the silver beastcoin door in Sea Serpent's Grotto (H-11) aggros through sneak/invis/deodorize.

## Expected Behavior
Brook Sahagin is supposed to aggro sound only.

## Current Behavior
Seems to have true sight/sound/smell.

## Relevant Wiki Link 
(IMPORTANT: AUGUST 2008 WIKI ENTRIES OR EARLIER)
https://ffxiclopedia.fandom.com/wiki/Brook_Sahagin

## Steps to Reproduce

1.
2.
3.
4.

## Context
This mob is guarding the passageway to complete the Utsusemi: Ichi scroll quest and is not passable without killing.

**Comment by Swerve256** [comment]
Cannot reproduce this issue. with a 25 rdm with snk and inv I get no aggro from Brook_Sahagins. Please contact me ingame to demonstrate the issue.

**Comment by zierk** [comment]
We had another incident of this happening today. The same spawn aggroed a 58 BRD through sneak. The mob despawned when the brd warped out because it was dragged out of its spawn range. When we waited for the next mob to spawn it did not aggro through sneak. We did not get the mob ID, but I suspect one or more of the lottery for that spawn point has a bug where it can see through sneak.

**Comment by JigokunoOusama** [comment]
I would make sure you didn't lose your buffs when opening the door.  Usually invisible, sneak, and deodorize drop when you trade the silver beastcoin.  The spawn is pretty close to the door, so it would be possible for it to aggro you immediately after trading the coin.

**Comment by zierk** [comment]
I did not see the buff icons removed when opening the door. Only invisible because you cannot trade while invisible. We have tried this a couple different ways. We have ensured sneak was active and got aggro. We have re-applied sneak inside the door and also got aggro. But as I said this happens, we kill the mob and then the respawn does not aggro you in the same manner. Also did not see this behavior with any other mobs along the way to the quest objective.
